tweets/robot-overlords.tweet

```diff
@@ -0,0 +1,3 @@
+I, for one, welcome our robot overlords 🤖
```

Comment:
maybe instead of "robot overlords" we can could say "robot friends"? Either works for me, feel free to ship it at your convenience :)

Reply:
I'm cool with robot friends!